neurom/features/section.py

```diff
@@ -41,26 +42,31 @@ def section_points(section):
     return section.points[:, COLS.XYZ]
 
 
+@cached_func()
 def section_path_length(section):
     """Path length from section to root."""
     return sum(s.length for s in section.iupstream())
 
 
+@cached_func()
 def section_length(section):
     """Length of a section."""
     return section.length
 
 
+@cached_func()
```

Comment:
I'm not sure the functions of this module should be cached since it is not possible to not use the cache, so it might lead to heavy memory consumption. Or maybe in this case we should set `maxsize` to a relevant value isntead of `maxsize=None` (i.e. 'infinite' cache).